src/main/java/salesforce/pages/LoginPage.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2020 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package salesforce.pages;
+
+import core.selenium.util.SalesForceGetProperties;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+/**
+ * LoginPage class.
+ *
+ * @author Oscar Lopez
+ * @version 1.0
+ */
+public class LoginPage extends BasePage {
+
+    /**
+     * Constant for the title of home page.
+     */
+    public static final String HOME_PAGE_TITLE = "Home Page ~ Salesforce - Developer Edition";
+
+    /**
+     * Web element for the user name text-box.
+     */
+    @FindBy(css = "input#username")
```

Comment:
Why do you use css locator for this web element?, maybe you could maintain this locator as "id=username"

Reply:
I tested if it works with that kind of CSS locator

Reply:
And works